Dockerfile

```diff
@@ -115,3 +116,5 @@ COPY "./dockerfiles/docker-entrypoint.sh" "/init"
 HEALTHCHECK --start-period=6m CMD "clamdcheck.sh"
 
 ENTRYPOINT [ "/init" ]
+
+USER clamav
```

Comment:
One of the use cases for a non-root clamav image is for deploying to kubernetes clusters that enforce `runAsNonRoot`. What I have experienced in testing this PR is the cluster I'm working with verifies if a user is non-root by looking at its UID and NOT the username. The specific error the cluster gives me is `container has runAsNonRoot and image has non-numeric user(clamav), cannot verify user is non-root`. I had to modify the above line to `USER 100` since `100` is the UID for user `clamav`. This allowed the container to spin up and work on the `runAsNonRoot` cluster I am working with. Is it possible to change this line to `USER 100` in this PR?